src/analysis/retail/monk/mistweaver/modules/core/HotAttributor.ts

```diff
@@ -41,18 +48,76 @@ class HotAttributor extends Analyzer {
       Events.applybuff.by(SELECTED_PLAYER).spell([SPELLS.ESSENCE_FONT_BUFF]),
       this.onApplyEF,
     );
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.RENEWING_MIST_HEAL),
+      this.onRemoveRem,
+    );
+  }
+
+  onRemoveRem(event: RemoveBuffEvent) {
+    debug &&
+      console.log(
+        'Removed rem from ' +
+          this.combatants.getEntity(event)?.name +
+          ' at ' +
+          this.owner.formatTimestamp(event.timestamp, 3),
+      );
   }
 
   onApplyRem(event: ApplyBuffEvent | RefreshBuffEvent) {
     if (this._hasAttribution(event)) {
-      return;
+      this.hotTracker.addAttributionFromApply(this.bouncedAttrib, event);
+      if (debug) {
```

Comment:
this debug statement is pretty messy, do we want to move it out to a utility function maybe?